pkg/webservice/routes.go

```diff
@@ -98,6 +98,14 @@ var routes = Routes{
 		GetContainerHistory,
 	},
 
+	// endpoint to set attributes of the logger
+	Route{
+		"Log",
+		"POST",
+		"/ws/v1/log",
+		ConfigureLogger,
+	},
+
```

Comment:
Its better have GET method also know what the set log level in YuniKorn.
Usually it ll be like GET the level, and then set it with desired value.
Thoughts. @wilfred-s @adamantal

Reply:
yes both should be implemented for completeness: 
- GET to retrieve
- PUT to update

POST is for creating a new object PUT is for updating an existing object.
See https://restfulapi.net/rest-put-vs-post/